tests/ops/integration_tests/test_integration_email.py

```diff
@@ -0,0 +1,141 @@
+        "email_dataset:payment": 0,
+        "postgres_example_test_dataset:customer": 0,
+        "postgres_example_test_dataset:report": 0,
+        "postgres_example_test_dataset:employee": 0,
+        "postgres_example_test_dataset:service_request": 0,
+        "postgres_example_test_dataset:visit": 0,
+        "postgres_example_test_dataset:orders": 0,
+        "postgres_example_test_dataset:login": 0,
+        "postgres_example_test_dataset:address": 0,
+        "postgres_example_test_dataset:payment_card": 0,
+        "email_dataset:daycare_customer": 0,
+        "postgres_example_test_dataset:order_item": 0,
+        "email_dataset:children": 0,
+        "postgres_example_test_dataset:product": 0,
+    }, "No data masked by Fidesops for the email collections"
+
+    raw_email_template_values = (
+        privacy_request.get_email_connector_template_contents_by_dataset(
+            CurrentStep.erasure, "email_dataset"
+        )
+    )
+
+    assert raw_email_template_values == {
+        "children": CollectionActionRequired(
+            step=CurrentStep.erasure,
+            collection=CollectionAddress("email_dataset", "children"),
+            action_needed=[
+                ManualAction(
+                    locators={
+                        "parent_id": ["email_dataset:daycare_customer:id"]
+                    },  # The only locator is on a separate collection on their end. We don't have data for it.
+                    get=None,
+                    update={
+                        "birthday": "null_rewrite",
+                        "first_name": "null_rewrite",
+                        "last_name": "null_rewrite",
+                        "report_card.grades": "null_rewrite",
+                        "report_card.behavior_issues": "null_rewrite",
+                        "report_card.disciplinary_action": "null_rewrite",
+                        "report_card.test_scores": "null_rewrite",
+                    },
+                )
+            ],
+        ),
+        "daycare_customer": CollectionActionRequired(
+            step=CurrentStep.erasure,
+            collection=CollectionAddress("email_dataset", "daycare_customer"),
+            action_needed=[
+                ManualAction(
+                    locators={
+                        "customer_id": [1]
+                    },  # We have some data from postgres they can use to locate the customer_id
+                    get=None,
+                    update={"scholarship": "null_rewrite"},
+                )
+            ],
+        ),
+        "payment": CollectionActionRequired(
+            step=CurrentStep.erasure,
+            collection=CollectionAddress("email_dataset", "payment"),
+            action_needed=[
+                ManualAction(
+                    locators={"payer_email": ["customer-1@example.com"]},
+                    get=None,
+                    update=None,  # Nothing to mask on this collection
+                )
+            ],
```

Comment:
Even though there's nothing to mask on the payment collection, I am still caching details for all the "email" collections because its locator may be useful in helping them find data on other downstream collections.